components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -603,4 +647,114 @@ - (void)didTapItemView:(UITapGestureRecognizer *)tap {
   }
 }
 
+- (void)applySelectionTemplateToSelectionViewForItemView:(UIView *)itemView {
+  if (!itemView) {
+    return;
```

Comment:
When would this be called with a nil view? 
Is that a likely scenario? 
Nit - If not, did you consider using a parameter assert?

Reply:
At the moment, I don't believe it is. It's just a bit of defensive coding to keep the code safer for the future. I don't think it should be an error if the user passes `nil` (the parameter is "null unspecified").  There may be a legitimate case of unselecting or "animating out" the selection indicator of an item is removed.